compiler/rustc_middle/src/mir/coverage.rs

```diff
@@ -135,7 +135,10 @@ impl Debug for CoverageKind {
                 "Expression({:?}) = {} {} {}",
                 id.index(),
                 lhs.index(),
-                if *op == Op::Add { "+" } else { "-" },
+                match op {
+                    Op::Add => "+",
+                    Op::Subtract => "-",
+                },
```

Comment:
a method for this is probably overkill, but maybe just implement `Display` for it?

Reply:
`Display` is a bit awkward, given that it will give as a `String` and we don't really need it... I'd just leave it as-is